src/ape_cache/schemas.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from pydantic import BaseModel
+
+"""
+Types to be fixed
```

Comment:
What does this comment mean? What is "fixed"?

Reply:
This was a note to me, should be a TODO
I'm not all the way there on the database design, so I left a note there as a reminder